gensim/scripts/segment_wiki.py

```diff
@@ -217,15 +225,8 @@ def get_texts_with_sections(self):
         # is dumb and would load the entire input into RAM at once...
         for group in utils.chunkize(page_xmls, chunksize=10 * self.processes, maxsize=1):
             for article_title, sections in pool.imap(segment, group):  # chunksize=10):
-                # article redirects and short stubs are pruned here
-                num_total_tokens = 0
-                for section_title, section_content in sections:
-                    if self.lemmatize:
-                        num_total_tokens += len(utils.lemmatize(section_content))
-                    else:
-                        num_total_tokens += len(tokenize(section_content))
-                if num_total_tokens < ARTICLE_MIN_WORDS or \
```

Comment:
Including redirects and stubs is a bad idea. That's typically (never?) what people want, out of Wikipedia dumps. 

We want to keep only meaningful articles, such as at least 500 plain text characters (~1 paragraph) or something.

Reply:
I don't think so (about short articles), because we provide parsed wikipedia dump "as-is" and short articles can be useful for users for special cases (and easy to filter later if needed), for this reason, I removed this part.

Reply:
But it's needed to filter trash (like the redirect), I'll add fix for this.

Reply:
Sounds good 👍